calculator-app/src/androidTest/java/camp/nextstep/edu/calculator/MainActivityTest.kt

```diff
@@ -244,4 +275,44 @@ class MainActivityTest {
         // Then
         onView(withId(R.id.textView)).check(matches(withText("3 +")))
     }
+
+    @Test
+    fun `메모리버튼을_누르면_현재계산식이_사라지고_기록화면이_보인다`() {
```

Comment:
다음과 같이 언더바를 제외하고 작성하셔도 됩니다!
```suggestion
    fun `메모리버튼을 누르면 현재계산식이 사라지고 기록화면이 보인다`() {
```
given-when-then 형식으로 작성해주신 것으로 보이는데 주석이 누락된 부분이 있는 것 같아요.